test/template/www/js/updateSync2x.js

```diff
@@ -0,0 +1,35 @@
+var app = {
+    // Application Constructor
+    initialize: function () {
+        this.bindEvents();
+    },
+    bindEvents: function () {
+        document.addEventListener('deviceready', this.onDeviceReady, false);
+    },
+    onDeviceReady: function () {
+        app.receivedDeviceReady();
+    },
+    // Update DOM on a Received Event
+    receivedDeviceReady: function () {
+        document.getElementById("deviceready").innerText = "Device is ready (scenario - update sync)";
+        console.log('Received Event: deviceready');
+        app.sendTestMessage("DEVICE_READY_AFTER_UPDATE");
+        /* invoke sync with UI options such that the update will not be installed */
```

Comment:
not sure, it was in the original updateSync.js and I didn't change it, we would need to change it there too